spec/views/user_management/show.html.erb_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe "user_management/show.html.erb", type: :view do
+  before(:each) do
+    @user_not_invited = create(:user)
+  end
+
+  it 'renders columns' do
```

Comment:
If you want to test what has been added, you must test all the three cases.

Reply:
I know, but I still don't know how to do it, could you help me?

Reply:
Sure, I'm going to ping you pvt.